Package.swift

```diff
@@ -30,6 +30,7 @@ let package = Package(
         .library(name: "NIOSSL", targets: ["NIOSSL"]),
         .executable(name: "NIOTLSServer", targets: ["NIOTLSServer"]),
         .executable(name: "NIOSSLHTTP1Client", targets: ["NIOSSLHTTP1Client"]),
+        .executable(name: "NIOSSLPerformanceTester", targets: ["NIOSSLPerformanceTester"]),
```

Comment:
Do we actually want to export this as a product? (https://github.com/apple/swift-nio/issues/1217)

Reply:
Ah well spotted, probably not.